app/build.gradle

```diff
@@ -20,19 +20,19 @@ dependencies {
     implementation 'com.jakewharton.timber:timber:4.5.1'
     implementation 'info.debatty:java-string-similarity:0.24'
     implementation 'com.borjabravo:readmoretextview:2.1.0'
-    implementation 'com.android.support.constraint:constraint-layout:1.0.2'
+    implementation 'com.android.support.constraint:constraint-layout:1.1.0'
     implementation ('com.mapbox.mapboxsdk:mapbox-android-sdk:5.4.1@aar'){
         transitive=true
     }
 
     implementation "com.github.deano2390:MaterialShowcaseView:1.2.0"
 
-    implementation "com.android.support:support-v4:$SUPPORT_LIB_VERSION"
-    implementation "com.android.support:appcompat-v7:$SUPPORT_LIB_VERSION"
-    implementation "com.android.support:design:$SUPPORT_LIB_VERSION"
-    implementation "com.android.support:customtabs:$SUPPORT_LIB_VERSION"
+    implementation "com.android.support:support-v4:27.1.1"
```

Comment:
You probably want to set `$SUPPORT_LIB_VERSION` to 27.1.1 instead of hardcoding all of it. ;)